kuma/wiki/kumascript.py

```diff
@@ -38,11 +38,19 @@ def should_use_rendered(doc, params, html=None):
             (force_macros or (not no_macros and not show_raw)))
 
 
-def _post(content, env_vars, timeout=config.KUMASCRIPT_TIMEOUT):
+def _post(content, env_vars, cache_control='',
```

Comment:
Super minor, but it's more common practice in this case to use `cache_control=None`.